src/main/java/io/github/hypixel_api_wrapper/wrapper/games/bedwars/HypixelBedWarsStats.java

```diff
@@ -0,0 +1,82 @@
+package io.github.hypixel_api_wrapper.wrapper.games.bedwars;
+
+import io.github.hypixel_api_wrapper.wrapper.games.HypixelGame;
+import org.json.JSONObject;
+
+public class HypixelBedWarsStats extends HypixelGame  {
```

Comment:
Well, first off that's just because we're only working on the BedWars classes right now as a prototype. I also kind of like the `HypixelBedWarsStats` name, because of the way it would be referenced. `HypixelBedWars = player.getGames().getBedWarsStats().getMode(HypixelBedWarsModes.SOLOS);`

Or, `HypixelBedWarsStats = player.getGames().getBedWarsStats();`